### ISSUE TITLE
Screenshot isn't saved / commited on Juno

### ISSUE THREAD

**Issue by terndrup** [elementary/screenshot#110]
So I've changed the default screenshot shortcuts for things like "Select an Area to Screenshot" and when I don't have the screenshot app running (i.e. it's not interactive in Plank) I can press the keyboard shortcut and it will come up with the area cursor and allow me to select an area. Upon selecting an area, nothing happens. The Screenshot window where you can view and choose to save your Screenshot does not come up.

Alternatively, if the Screenshot app is already running interactively in my Plank, invoking the shortcut and selecting an area does work correctly.

tl;dr: Pressing the shortcut doesn't save the screenshot after selecting an area, only if the Screenshot is already fired up.

This can't be by design, can it?

**Comment by peteruithoven** [comment]
Are you sure nothing happens? 
- There is no shutter sound?
- There is no screenshot added to `Pictures` or `Pictures/Screenshots/`?

**Comment by terndrup** [comment]
There is a shutter sound, and it seems to some times add the screenshot to my "Pictures" folder.
But the UI doesn't come up to allow me to choose a location or to copy it to clipboard (which is what I really want).

Is there a way to set it to always copy the latest taken screenshot to clipboard? Kinda like specifying an "After Screenshot taken" action. @peteruithoven

**Comment by peteruithoven** [comment]
I think that's by design. 
Actually technically the keyboard shortcut screenshot taking is handled by [Gala](https://github.com/elementary/gala/) instead of the [Screenshot](https://github.com/elementary/screenshot) application. (Sort of relevant: https://github.com/elementary/screenshot/issues/103)

If you want to customize what the shortcut does I'd suggest adding a custom shortcut to using the screenshot application as commandline utility.
Depending on whether you are on Loki or Juno you can find info here:
```
io.elementary.screenshot-tool --help
```
or
```
screenshot-tool --help
```

**Comment by terndrup** [comment]
Mhm so I just tried that. I wanted to add the -c flag but it does not save the file anywhere by just running that. I'll mess around with it and see if I can get it to work.

Thanks for your help!

**Comment by danrabbit** [comment]
Closing this issue since when using keyboard shortcuts the screenshot is taken by Gala and not by Screenshot
